Allow label update

JackWilb opened the pull request:
### Does this PR close any open issues?
No. We hadn't opened one for it

### Give a longer description of what this PR addresses and why it's needed
Add the ability to toggle the label variable between any of the node attributes. If no label is selected, the `_key` is shown

### Provide pictures/videos of the behavior before and after these changes (optional)
Without a variable selected:
![image](https://user-images.githubusercontent.com/36867477/152251144-a0f06fcf-22f2-40d5-bc3a-77ae487b502f.png)

With a variable selected:
![image](https://user-images.githubusercontent.com/36867477/152251177-a72cb76f-24d9-4d25-a499-5568280c2e27.png)

### Are there any additional TODOs before this PR is ready to go?

TODO:
- [x] Make an issue to refactor controls. I plan to make it more like the multilink one (#357)

gotdairyya reviewed (dismissed):
Looks good (and happy for this functionality). It does not play well with the aggregated node types though:
- the label is funky after nodes are aggregated
- the label doesn't show immediately if a label was chosen but then the nodes are aggregated

I will go ahead and approve, let me know if you want me to create issues for this.

JackWilb commented:
> Looks good (and happy for this functionality). It does not play well with the aggregated node types though:

I think I've addressed some of that. Would you mind double checking it? I updated it so that when you aggregate, the variable you aggregate by is chosen as the label. I have some ideas for how we might allow different values for the supernodes and the children.

> I will go ahead and approve, let me know if you want me to create issues for this.

I'll create an issue for labeling with different values for children and supernodes once you check my changes.

gotdairyya reviewed (approved):
Works great!
